### PR TITLE
Add support for external minio object stores

### PR DESCRIPTION
**Pull request by sthapa**
PR adds support for using an external minio object store

### CODE REVIEW

**Comment by ivukotic** [comment]
I just tested it and it works.

**Review by ivukotic — APPROVED** [review]
tested fine

File: servicex/__init__.py [hunk 1]
```diff
@@ -86,9 +86,17 @@ def create_app(test_config=None,
 
         if app.config['OBJECT_STORE_ENABLED']:
             if not provided_object_store:
+                if 'MINIO_SECURED' in app.config:
+                    if isinstance(app.config['MINIO_SECURED'], bool):
+                        use_https = app.config['MINIO_SECURED']
+                    else:
+                        use_https = strtobool(app.config['MINIO_SECURED'])
```
Comment (BenGalewsky) [inline servicex/__init__.py:93]: Why is this needed? Shouldn't the Helm chart guarantee this is a boolean?

File: servicex/transformer_manager.py [hunk 2]
```diff
@@ -108,8 +114,11 @@ def create_job_object(request_id, image, chunk_size, rabbitmq_uri, workers,
                 client.V1EnvVar(name='MINIO_ACCESS_KEY',
                                 value=current_app.config['MINIO_ACCESS_KEY']),
                 client.V1EnvVar(name='MINIO_SECRET_KEY',
-                                value=current_app.config['MINIO_SECRET_KEY']),
+                                value=str(current_app.config['MINIO_SECRET_KEY'])),
```
Comment (BenGalewsky) [inline servicex/transformer_manager.py]: What type can the secret key if not str already?

File: servicex/__init__.py [hunk 3]
```diff
@@ -86,9 +86,17 @@ def create_app(test_config=None,
 
         if app.config['OBJECT_STORE_ENABLED']:
             if not provided_object_store:
+                if 'MINIO_SECURED' in app.config:
+                    if isinstance(app.config['MINIO_SECURED'], bool):
```
Comment (BenGalewsky) [inline servicex/__init__.py:90]: Please update `app.conf.template` to include a default setting for desktop debugging of the app.

File: servicex/transformer_manager.py (hunk 2 above)
Comment (sthapa) [reply servicex/transformer_manager.py]: Yeah, that's a stray change from when I was debugging an issue.  I've fixed that.

File: servicex/__init__.py (hunk 3 above)
Comment (sthapa) [reply servicex/__init__.py:90]: Already there

File: servicex/__init__.py (hunk 1 above)
Comment (sthapa) [reply servicex/__init__.py:93]: I believe the Helm chart should guarantee this but I believe I ran into an error due to this so I added a check

**Review by BenGalewsky — APPROVED** [review]
Ok
